cicd/gitops/setting-up-argocd-instance.adoc

```diff
@@ -0,0 +1,25 @@
+:_content-type: ASSEMBLY
+[id="setting-up-argocd-instance"]
+= Setting up a new Argo CD instance
+include::_attributes/common-attributes.adoc[]
+:context: setting-up-argocd-instance
+
+toc::[]
+
+By default, the {gitops-title} installs an instance of Argo CD in the `openshift-gitops` namespace with additional permissions, which allows to manage certain cluster scoped resources.
+
+You can install and deploy a new Argo CD instance for managing cluster configurations or deploying applications. By default, any new instance will only have permissions to manage resources in the namespace where it is deployed.
```

Comment:
```suggestion
To manage cluster configurations or deploy applications, you can install and deploy a new Argo CD instance. By default, any new instance only has permissions to manage resources in the namespace where it is deployed.
```
- Where possible, avoid the future-tense. Use the present tense instead. 
- What is the intention of this paragraph? Are you describing the user's motivation for (re)installing Argo CD instead of using the default installation? If so, you can emphasize that motivation by putting it first.